auth0/resource_auth0_log_stream.go

```diff
@@ -127,8 +127,11 @@ func newLogStream() *schema.Resource {
 							RequiredWith: []string{"sink.0.http_content_format", "sink.0.http_endpoint", "sink.0.http_content_type"},
 						},
 						"http_custom_headers": {
-							Type:        schema.TypeSet,
-							Elem:        &schema.Schema{Type: schema.TypeString},
+							Type: schema.TypeList,
```

Comment:
By changing this to a TypeList we are now saying that the order of the header collection actually matters. Any particular reasons why we're doing this?